test/integration/detailed_guide_test.rb

```diff
@@ -48,4 +48,25 @@ class DetailedGuideTest < ActionDispatch::IntegrationTest
       assert page.has_text?('This guidance was published under the 2010 to 2015 Conservative and Liberal Democrat coalition government')
     end
   end
+
+  test 'detailed guide that only applies to a set of nations' do
+    setup_and_visit_content_item('national_applicability_detailed_guide')
+
+    within shared_component_selector('metadata') do
```

Comment:
I had a comment about that and I removed it; but no.

That helper function purposefully removes `other`: `args.merge!(args.delete("other")) if args.key?("other")`.

I could create another helper function [but didn't seem worth it for 2 uses](http://c2.com/cgi/wiki?ThreeStrikesAndYouRefactor).